### ISSUE TITLE
The Tab contol type of the General/Feedback Tab elements are  announced as Button in the Xamarin Inspector application.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#296]
### Pre-requisites:
Launch the Visual Studio for Mac and Sign-in with Enterprise account.

Create projects:

1. Go to Visual Studio-> File-> New Solution. -> Multiplatform-> App-> Xamarin.Forms-> Blank Forms App.
2. Hit on Next button to create the project.
3. Hit on Debug(Play)/Build button to build/deploy the solution/project.
4. As per the choice/requirements, Change the Device platform like iOS/ Android and iPhone X iOS 11.3 etc.
5. Hit on Inspector icon button (i.e., in the top toolbar buttons of the VS) to inspect the project.
6. On hitting on inspector button, Xamarin Inspector is Launched.

### Repro-Steps:
1. Follow the Pre-requisites to Launch the Xamarin Inspector from Visual Studio, Turn On the VoiceOver.
2. Navigate to Xamarin Inspector-> preferences->Hit On VO keys (the Control and Option keys) + space bar, which opens the General tab by default.
3. Navigate to these General and feedback tabs using VO+ arrow keys.
5. Observe the VoiceOver announcements for these Genaral and Feedback tabs.
### Expected Behavior
The VoiceOver should announce as General/Feedback Tab instaed of General/Feedback button.
### Actual Behavior
The VoiceOver is announcing as button for these General and feedback tabs using VO+ arrow keys.
### User Impact
The Screenreader user won't be able to know the exact purpose/function of the these elements until control type is not correct.

![vo2 1_prefrencegeneraltabcontrol](https://user-images.githubusercontent.com/22956130/39988821-0471caac-5786-11e8-8ad4-f0161dcfa065.png)
![vo2 2_prefrencefeedbacktabcontrol png](https://user-images.githubusercontent.com/22956130/39988822-04a8e00a-5786-11e8-9e2c-ad6b004dd119.png)

### Environment
Visual Studio for Mac Enterprise [Version: 7.5 Preview (7.5 build 1222)]
Xamarin Inspector [Version: 1.4 (master/b3f92f9)]

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830
